bcipy/helpers/task.py

```diff
@@ -370,186 +373,153 @@ def get_key_press(
     return None
 
 
+def data_reshaper(input_data_type: InputDataType, *args, **kwargs):
+    if input_data_type == InputDataType.TRIAL:
+        return trial_reshaper(*args, **kwargs)
+    elif input_data_type == InputDataType.INQUIRY:
+        return inquiry_reshaper(*args, **kwargs)
+    else:
+        raise NotImplementedError(f"Unsupported input data type: {input_data_type}")
+
+
+def grouper(iterable, n, fillvalue=None):
+    args = [iter(iterable)] * n
+    return zip_longest(*args, fillvalue=fillvalue)
+
+
+def inquiry_reshaper(trial_target_info: list,
+                     timing_info: list,
+                     eeg_data: np.ndarray,
+                     fs: int,
+                     trials_per_inquiry: int,
+                     offset: float = 0,
+                     channel_map: List[int] = DEFAULT_CHANNEL_MAP,
+                     trial_length: float = 0.5):
+    """Extract inquiry data and labels.
+    Args:
+        trial_target_info (list): labels each trial as "target", "non-target", "first_pres_target", etc
+        timing_info (list): Timestamp of each event in seconds
+        eeg_data (np.ndarray): shape (channels, samples) preprocessed EEG data
+        fs (int): sample rate of preprocessed EEG data
```

Comment:
It does not have to be preprocessed

Reply:
True - I intended to convey "provide the actual sample rate, rather than the original sample rate". I'll clarify the doc